README.md

```diff
@@ -6,8 +6,34 @@ This package uses `reticulate` to wrap `challengeutils` to create the Synapse R
 
 ## Installation
 
-*NOTE*: This package will not work with `synapser`.  Do not load `library(synapser)`
+This package requires the an installation of Python > 3.6 and `challengeutils`. *NOTE*: This package will not work with `synapser`.  Do not load `library(synapser)`
 
+
+In command line, we recommend you using anaconda.  Instructions to install anaconda can be found [here](https://docs.anaconda.com/anaconda/install/)
+
+```
+conda create -n challenge python=3.7
+pip install challengeutils
+```
+
+In R:
+
+```
+devtools::install_github("Sage-Bionetworks/challengerutils")
 ```
-devtools::install_github("Sage-Bionetworks/challengescoring")
+
+
+## Usage
+
+The functions will mimic the Python implementation of `challengeutils`.  PLease see the [documentation here](https://sage-bionetworks.github.io/challengeutils/).
+
+* Query Evaluation Queue
+```
+library(challengerutils)
+use_condaenv('challenge')
```

Comment:
Might be good to add `library(reticulate)` so that `use_condaenv('challenge')` works.

Either that or specify somewhere that the `reticulate` has already been loaded.

Reply:
No need to do that.  Because `reticulate` is loaded.

Reply:
Right right, forgot this was going to be a package, not just lines of code.